types/promise-sftp/promise-sftp-tests.ts

```diff
@@ -0,0 +1,9 @@
+import * as PromiseSftp from "promise-sftp";
+
+const sftp = new PromiseSftp();
```

Comment:
Since `PromiseSftp` is a class, that is not supported by ES module, it should be `import = require` here https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports